drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -28,14 +28,18 @@ LOG_MODULE_REGISTER(esp32_wifi, CONFIG_WIFI_LOG_LEVEL);
 
 /* use global iface pointer to support any ethernet driver */
 /* necessary for wifi callback functions */
-static struct net_if *esp32_wifi_iface;
+static struct net_if *esp32_wifi_iface = NULL;
 static struct esp32_wifi_runtime esp32_data;
 
+static struct net_if *esp32_wifi_iface_ap = NULL;
+static struct esp32_wifi_runtime esp32_ap_mode_data;
```

Comment:
`esp32_ap_mode_data` should this be renamed to `esp32_ap_sta_data` instead to reflect what it really is?

Reply:
WDYT about renaming it to `esp32_ap_data`. As this is used for AP interface in AP+STA mode.